#
# OpenIndiana specific patch to provide support for configuration
# fragment files by default.
# This is intentionally separate from any other patch that modifies
# sshd_config so that over time those changes may be able to be delivered
# as fragement files rather than patching the source.
--- openssh-10.0p1/ssh_config.orig	2025-04-09 09:02:43.000000000 +0200
+++ openssh-10.0p1/ssh_config	2025-04-16 20:03:57.440742446 +0200
@@ -44,3 +44,5 @@
 #   ProxyCommand ssh -q -W %h:%p gateway.example.com
 #   RekeyLimit 1G 1h
 #   UserKnownHostsFile ~/.ssh/known_hosts.d/%k
+#
+    Include /etc/ssh/ssh_config.d/*.conf
--- openssh-10.0p1/sshd_config.orig	2025-04-16 19:59:46.371399072 +0200
+++ openssh-10.0p1/sshd_config	2025-04-16 20:03:39.111524644 +0200
@@ -120,3 +120,5 @@
 #	AllowTcpForwarding no
 #	PermitTTY no
 #	ForceCommand cvs server
+#
+Include /etc/ssh/sshd_config.d/*.conf
